test/testcafe/spec/EnrollAuthenticatorOktaVerify_spec.js

```diff
@@ -860,13 +865,36 @@ test
 
 test
   .meta('v3', false)
-  .requestHooks(logger, enrollDeviceBootstrapMocks)('should be able to see OV device bootstrap enrollment instructions without polling', async t => {
+  .requestHooks(logger, enrollDeviceBootstrapMocks)('should be able to see OV device bootstrap enrollment instructions without polling with one device', async t => {
+    const enrollOktaVerifyPage = await setup(t);
+    await checkA11y(t);
+    await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
+    await t.expect(enrollOktaVerifyPage.getSubHeader()).eql(deviceBootstrapSubtitle);
+    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(0)).eql(deviceBootstrapInstruction1);
+    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(1)).eql(deviceBootstrapInstruction2);
+    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(2)).eql(deviceBootstrapInstruction3);
+    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(3)).eql(deviceBootstrapInstruction4);
+    await t.expect(await enrollOktaVerifyPage.getClosingText()).eql(deviceBootstrapClosing);
+
+    await t.expect(enrollOktaVerifyPage.getTryDifferentWayText().exists).notOk();
+    await t.expect(await enrollOktaVerifyPage.returnToAuthenticatorListLinkExists()).ok();
+    await t.expect(await enrollOktaVerifyPage.signoutLinkExists()).ok();
+
+    // expect no polling for device bootstrap page
+    await t.expect(logger.count(
+      record => record.response.statusCode === 200 &&
+      record.request.url.match(/poll/)
+    )).eql(0);
+  });
+
+test
+  .meta('v3', false)
+  .requestHooks(logger, enrollDeviceBootstrapMocksMultipleDevices)('should be able to see OV device bootstrap enrollment instructions without polling with multiple devices', async t => {
     const enrollOktaVerifyPage = await setup(t);
     await checkA11y(t);
     await t.expect(enrollOktaVerifyPage.getFormTitle()).eql('Set up Okta Verify');
     await t.expect(enrollOktaVerifyPage.getSubHeader()).eql(deviceBootstrapSubtitle);
-    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(0)).contains(deviceBootstrapInstruction1);
-    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(0)).contains('testDevice1');
+    await t.expect(await enrollOktaVerifyPage.getNthInstructionBulletPoint(0)).eql(deviceBootstrapInstruction1);
```

Comment:
pls assert we don't have device2 name

Reply:
sure, added the assertion: https://github.com/okta/okta-signin-widget/pull/3362/commits/cf146c07a86f4f40973071cbd1673aba00ab1ce0
though on this line we assert equality to `Open Okta Verify on any of your other Okta Verify devices (Such as your testDevice1).`